### ISSUE TITLE
Database

### ISSUE THREAD

**Issue by cdmarioe** [schollz/find#165]
Hi guys,
I have been testing Find running on docker and it's work great, but i would like to know if it's possible, where FIND stores data of the devices..if i can access through mysql....Thank you so much, and sorry for my english....

**Comment by schollz** [comment]
You cannot access through mysql. The database is a Bolt database. You can export it with FIND as JSON. Download the DB and then `find -dump GROUPNAME`.

**Comment by cdmarioe** [comment]
Any other way to watch the content?

**Comment by schollz** [comment]
Yes, you can access it with Bolt tools, see https://github.com/boltdb/boltd
